websites/A/AnimeDigitalNetwork/presence.ts

```diff
@@ -7,21 +7,29 @@ const presence = new Presence({
   });
 
 presence.on("UpdateData", async () => {
-  const video: HTMLVideoElement = document.querySelector("video.vjs-tech"),
+  const video: HTMLVideoElement = document.querySelector(
+      "#adn-video-js_html5_api"
+    ),
     presenceData: PresenceData = {
       largeImageKey: "logo"
     };
 
   if (document.location.pathname.includes("video") && video) {
-    if (video && !isNaN(video.duration)) {
+    if (!isNaN(video.duration)) {
       const title = document.querySelector(
-          "#root > div > div > div.sc-pbWVv.hTvDIL > div > div:nth-child(1) > div.sc-kbKFCX.sc-kgMcbC.kecmmo > div:nth-child(1) > div > div > h1 > a"
-        ),
-        timestamps = presence.getTimestamps(
-          Math.floor(video.currentTime),
-          Math.floor(video.duration)
-        );
-      presenceData.details = title.textContent;
+          'meta[name="description"]'
+        ) as HTMLMetaElement,
+        title2 = title.content.replace(/[0-9]/g, " ").substr(6),
+        title3 = title2.length - 131;
+      if (title3 < 0) {
+        const title4 = title2.length - 112;
+        presenceData.details = title2.slice(0, title4);
+      } else presenceData.details = title2.slice(0, title3);
+
+      const timestamps = presence.getTimestamps(
+        Math.floor(video.currentTime),
+        Math.floor(video.duration)
```

Comment:
dont think you need to math floor it, and you could use getVideoTimestamps (or its called something like that at least)

Reply:
![image](https://user-images.githubusercontent.com/9348108/125168762-54987e00-e1a7-11eb-82a6-584daf7a71ea.png)
ofc you need to